packages/ethereum/deploy/05_HoprChannels.ts

```diff
@@ -19,11 +19,21 @@ const main: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const hoprToken = await deployments.get('HoprToken')
 
-  await deployments.deploy('HoprChannels', {
+  let salt: string
```

Comment:
how is this related to versioning epic?

Reply:
it is not directly related but makes things easier and less confusing when having multiple instances of the same contract on many chains

Reply:
I dont see how this helps development, just extra lines of code

Reply:
Originally, I wanted to make all deployments of all contract deterministic but this only works for HoprChannels without major channels.

The benefit that I see is that you cannot redeploy a contract with the same environment on the same chain. It now forces you to specify a *different* environment.
Furthermore, you can easily check if the specific version of `HoprChannels` has been deployed to a given network. Etherscan also seems to support checking for one contract on multiple chains, see https://blockscan.com/address/0x6F80d1a3AB9006548c2fBb180879b87364D63Bf7